acdc-core/src/main/resources/reference.conf

```diff
@@ -6,17 +6,17 @@ com.salesforce.mce.acdc {
             // Simple datasource with no connection pooling. The connection pool has already been
             // specified with HikariCP.
             dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-            properties = {
-              serverName = "db"
-              serverName = ${?POSTGRES_SERVER}
-              portNumber = "5432"
-              portNumber = ${?POSTGRES_PORT}
-              databaseName = "postgres"
-              databaseName = ${?POSTGRES_DBNAME}
-              user = "postgres"
-              user = ${?POSTGRES_USER}
-              password = ${?POSTGRES_PASSWORD}
-            }
+
```

Comment:
@thomaschangsf why delete `properties`?

Reply:
The db connector instantiation, in my local environment, fails to read the db parameters when it is under properties.